src/components/PatchDeleteSprite.jsx

```diff
@@ -5,11 +5,11 @@ import Grid from '@mui/material/Grid';
 import DeleteOutlineIcon from '@mui/icons-material/DeleteOutline';
 
 export function PyatchDeleteSprite(props) {
-    const { editingTargetId, onDeleteSprite } = useContext(pyatchContext);
+    const { targetIds, editingTargetId, onDeleteSprite } = useContext(pyatchContext);
 
-    return(
+    return (
         <Grid container justifyContent="center">
-            <Button variant="contained" onClick={()=>onDeleteSprite(editingTargetId)} sx={{m:"1vh"}}><DeleteOutlineIcon/></Button>
+            <Button variant="contained" disabled={targetIds[0] == editingTargetId} onClick={() => { if (targetIds[0] != editingTargetId) { onDeleteSprite(editingTargetId); } }} sx={{ m: "1vh" }}><DeleteOutlineIcon /></Button>
```

Comment:
If you could use the `isStage` prop of the target class for this disabled parameter that would be great. Example of its use below.